### PR TITLE
Implement Account Creation Header

### LINKED ISSUE #360: Account creation header + profile photo
### Summary and Background
Account creation header + profile photo for admins and volunteers.

### User Story
As an Admin/Volunteer, I want to upload/remove profile photos to my account.

### Solution (Core Elements)
Create header + profile photo for account creation personal info page, accessed from account creation landing page (#340). On the same page is the personal info fields (#341).

Profile photo can be uploaded and removed. If uploaded, the photo will replace the profile icon on the page. If removed, the default icon will reappear.

If a photo is submitted, it will be appear at the profile location on the navbar when the admin/volunteer is logged in.

This ticket has no difference between admins and volunteers.

### Mockups

Other designs: 
https://www.figma.com/file/tOK6AjkaRZ1NBXo191D64C/Sistering-%2F-User-Authentication-Process?node-id=390%3A1975

## Technical
* In the component `frontend/src/components/pages/NewAccountPage.tsx`, create buttons for uploading and removing a photo
* Create the placeholder (and its label) for the profile photo

### PR DESCRIPTION
**Pull request by AdrianaCeric**
## Ticket link

Closes #360 

## Implementation description
* Added placeholder profile image
* Added two buttons ("Upload Photo" and "Remove Photo")
* Added State to the profile so it can update

## Steps to test
1. Login to application
2. View the header of the main page

## What should reviewers focus on?
* Is the styling okay?
* Should I implement more functionality for buttons in this ticket?

## Checklist
- [X] My PR name is descriptive and in imperative tense
- [X] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [X] I have run the appropriate linter(s)
- [X] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

## Screenshot
image

### CODE REVIEW

**Comment by MatoPlus** [comment]
This might have a merge conflict with #351 :o

File: frontend/src/components/pages/Default.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import AuthContext from "../../contexts/AuthContext";
 import { Role } from "../../types/AuthTypes";
 import { PostingResponseDTO } from "../../types/api/PostingTypes";
 import PostingCard from "../volunteer/PostingCard";
+import NewAccountPage from "./NewAccountPage";
```
Comment (MatoPlus) [inline frontend/src/components/pages/Default.tsx:13]: Looks like there are still some unnecessary changes here. Please run `yarn fix` in the backend folder to remove all the linting errors.

Comment (AdrianaCeric) [reply frontend/src/components/pages/Default.tsx:13]: I got this when I run `yarn fix` in the backend. what do i do?
image

**Review by MatoPlus — APPROVED** [review]
LGTM
